### PR TITLE
chore(deps): bumping cdk from 1.108.1 to 1.111.0

### PR DESCRIPTION
**Pull request by jusiskin**
## Notes

Bumps CDK version from `1.108.1` &rarr; `1.111.0`

## Testing

*   Built RFDK
*   Deployed the basic example app successfully
    *   Connected a Deadline Monitor to the farm successfully
    *   Submitted a command-line ping job successfully
    *   Ping job rendered tasks successfully

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/aws-rfdk/package.json [hunk 1]
```diff
@@ -65,150 +65,150 @@
...
+    "@aws-cdk/region-info": "1.111.0",
     "constructs": "^3.3.69"
   },
   "engines": {
     "node": ">= 12.18.3 <13 || >=13.7.0"
   },
   "stability": "stable",
   "maturity": "stable"
-}
+}
```
Comment (jericht) [inline packages/aws-rfdk/package.json]: nit: looks like we're missing a new line here

**Review by jericht — CHANGES_REQUESTED** [review]

File: packages/aws-rfdk/package.json (hunk 1 above)
Comment (jusiskin) [reply packages/aws-rfdk/package.json]: fixed

**Review by jericht — DISMISSED** [review]

**Review by ddneilson — CHANGES_REQUESTED** [review]
Looks like there are some new peer dependencies that we'll need to add:
```
 [3/4] Linking dependencies...
warning " > @aws-cdk/aws-sns@1.111.0" has unmet peer dependency "@aws-cdk/aws-codestarnotifications@1.111.0".
warning " > @aws-cdk/aws-codebuild@1.111.0" has unmet peer dependency "@aws-cdk/aws-codestarnotifications@1.111.0".
warning " > @aws-cdk/aws-codepipeline@1.111.0" has unmet peer dependency "@aws-cdk/aws-codestarnotifications@1.111.0".
```

**Comment by jusiskin** [comment]
> Looks like there are some new peer dependencies that we'll need to add:
> 
> ```
>  [3/4] Linking dependencies...
> warning " > @aws-cdk/aws-sns@1.111.0" has unmet peer dependency "@aws-cdk/aws-codestarnotifications@1.111.0".
> warning " > @aws-cdk/aws-codebuild@1.111.0" has unmet peer dependency "@aws-cdk/aws-codestarnotifications@1.111.0".
> warning " > @aws-cdk/aws-codepipeline@1.111.0" has unmet peer dependency "@aws-cdk/aws-codestarnotifications@1.111.0".
> ```

Good catch. This is now fixed in https://github.com/aws/aws-rfdk/pull/500/commits/36115d34e1f7b1167dc88cddce9119e0b1d6a6dd

File: packages/aws-rfdk/package.json [hunk 2]
```diff
@@ -65,145 +65,146 @@
...
+    "@aws-cdk/aws-batch": "1.111.0",
+    "@aws-cdk/aws-certificatemanager": "1.111.0",
+    "@aws-cdk/aws-cloudformation": "1.111.0",
+    "@aws-cdk/aws-cloudfront": "1.111.0",
+    "@aws-cdk/aws-cloudwatch": "1.111.0",
+    "@aws-cdk/aws-cloudwatch-actions": "1.111.0",
+    "@aws-cdk/aws-codebuild": "1.111.0",
+    "@aws-cdk/aws-codecommit": "1.111.0",
+    "@aws-cdk/aws-codeguruprofiler": "1.111.0",
+    "@aws-cdk/aws-codepipeline": "1.111.0",
```
Comment (ddneilson) [inline packages/aws-rfdk/package.json:97]: Missed adding `@aws-cdk/aws-codestarnotifications": "1.111.0",` here. deps & peer deps are in-sync.

**Review by ddneilson — CHANGES_REQUESTED** [review]

File: packages/aws-rfdk/package.json (hunk 2 above)
Comment (jusiskin) [reply packages/aws-rfdk/package.json:97]: Fixed

**Review by ddneilson — APPROVED** [review]

**Review by jericht — APPROVED** [review]

**Pull request merged**